web/cypress/integration/route/create-route-with-upstream.spec.js

```diff
@@ -25,6 +25,7 @@ context('Create Route with Upstream', () => {
     description: '#desc',
     upstreamSelector: '#upstream_id',
     searchName: '[title=Name]',
+    custom: '[title=Custom]',
```

Comment:
the meaning of `custom` is not very clear IMHO, it would be better to change a name, e.g: something like `select_upstream`  ^_^

Reply:
Thanks, I agree with you